upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -709,6 +682,17 @@ func (c *ApplyClusterCmd) Run() error {
 		// Avoid making changes on a dry-run
 		shouldPrecreateDNS = false
 
+	case TargetInventory:
+		// Utilized to build an api.Inventory
+		// This target was created because TargetDryRun outputs to STDOUT.  This allows for the rendering of
```

Comment:
After you made the changes for assets, I mentioned that we probably still want to use another target.  If I remember correctly you agreed. 

From my perspective, the output from a dryrun does apply to the output of the inventory of a cluster.  Too much information at times is worse than not enough.